react-sdk/src/elements/GooglePayButtonElement.tsx

```diff
@@ -0,0 +1,170 @@
+import React, { RefObject, useEffect, useRef } from 'react'
+import * as Types from '../types'
+import { PublicSquare } from '@publicsquare/elements-js/PublicSquare'
+import { PublicSquareInitOptions } from '../types'
+import { GooglePayConfiguration } from '@publicsquare/elements-js/types'
+
+const GooglePayButtonElement: React.FC<Types.GooglePayButtonElementProps> = ({
+  id,
+  environment = 'TEST',
+  merchantId = '12345678901234567890',
```

Comment:
Similar comments as the JS stuff. I think we want to require them to set some of these. We definitely don't want to create defaults for transaction info, merchant ID, and merchant name. We can set default card networks, but it should move to a configuration option.